WORKSPACE

```diff
@@ -6,5 +6,5 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
      name = "com_google_googletest",
      urls = ["https://github.com/google/googletest/archive/master.zip"],
```

Comment:
Should this be changed to main.zip as well?

Reply:
There was no problem even if I used it as master.zip in the `urls`.
When I checked, it redirected to main.zip.
There is no problem with that line, but it would be better to change it.